tlsr9/ble/algorithm/crypto/crypto_alg.h

```diff
@@ -41,6 +41,22 @@ typedef struct{
 	unsigned char mac[16];
```

Comment:
Thats common file for b91 & b92. Don't we have an issues on b91?

Reply:
These changes apply only to B92